src/main/views/courts/tabs/typesContent.njk

```diff
@@ -0,0 +1,143 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+<h2 class="govuk-heading-l">Court Types and Codes</h2>
+
+{% if errorMsg != '' %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: [
+      {
+        text: errorMsg
+      }
+    ]
+  }) }}
+{% endif %}
+
+{% if updated === true %}
+  {{ govukPanel({
+    titleText: "Court Types updated"
+  }) }}
+{% endif %}
+
+
+{% for item in items %}
+
+{% if item.text === "Magistrates' Court"  and item.checked %}
+
+{% endif %}
+
+
+{% if item.text === "Magistrates' Court"  %}
+
+ {% set magistratesCodeError = false %}
+  {% if item.checked and item.code | valid === false %}
+   {% set magistratesCodeError = { text: 'Magistrates code is required and must be numeric.' } %}
+  {% endif %}
+
+{% set magistratesCourtCodeHtml %}
+{{ govukInput({
+ label: {
+    text: "Magistrates' Court Code"
+  },
+  classes: "govuk-!-width-one-third",
+  id: "magistratesCourtCode",
+  name: "magistratesCourtCode",
+  type: "text",
+  value: item.code,
+  errorMessage: magistratesCodeError
+
+}) }}
+{% endset -%}
+
+{% set item = item | setAttribute('conditional',{ html: magistratesCourtCodeHtml }) %}
+
+{% endif %}
+
+
+
+
+{% if item.text === "County Court" %}
+
+{% set countyCourtCode = false %}
+ {% if item.checked and  item.code | valid === false  %}
+   {% set countyCourtCode = { text: 'County code is required and must be numeric.' } %}
+ {% endif %}
+
+{% set countyCourtCodeHtml %}
+{{ govukInput({
+  id: "countyCourtCode",
```

Comment:
All of the inputs on this page are contained in a loop. Could we end up with more than one input having the same ID?

Reply:
only 3 input are in loop with unique ID